docs/index.rst

```diff
@@ -0,0 +1,29 @@
+Welcome to plankton's documentation!
+====================================
+
+Plankton is a Python package that makes it easy to develop complex stateful device simulations. It
+is licensed under GPL version 3 and the source is available on github_, where you are welcome to
+open new issues so the package can improve.
+
+If you're new to Plankton reading the :ref:`user_guide` is a good start. Once you plan on writing
+your own device it's time to go through the :ref:`developer_guide`. The :ref:`api_documentation`
```

Comment:
Not sure if this is worth doing anything about, but these generate links with anchors (e.g.: `api/index.html#api-documentation`) when really what is intended is just linking to the page. In the TOC below, the links appear without the anchor. This has the side-effect of fooling a browser into not showing both as visited unless the user has actually clicked both version, even though they link to the same thing.

One way to fix this would be to use `:doc:` instead of `:ref:`:

```
:doc:`developer_guide/index`
```
But that is a bit more verbose. So it's a question of which bugs us more...